packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -3922,6 +4001,20 @@ class _MenuDefaultsM3 extends MenuStyle {
 
   @override
   VisualDensity get visualDensity => Theme.of(context).visualDensity;
+
+  @override
+  AnimationStyle get sizeAnimationStyle {
+    return switch (Theme.of(context).platform) {
+      TargetPlatform.iOS || TargetPlatform.android || TargetPlatform.fuchsia
+        => AnimationStyle(
+          curve: Curves.easeInOutCubicEmphasized,
+          reverseCurve: Curves.easeInOutCubicEmphasized.flipped,
+          duration: const Duration(milliseconds: 500),
+        ),
+      TargetPlatform.macOS || TargetPlatform.linux || TargetPlatform.windows
+        => AnimationStyle.noAnimation,
```

Comment:
Is it clarified in a spec that desktop dropdown menus should have no animation?

Reply:
No. We don't have specs for desktops. Initially we don't have any animation on menus. Here is just to follow the original behavior:)

Reply:
Can you doc this? Probably just exactly what you said here.